src/services/sync-checker.ts

```diff
@@ -190,14 +190,26 @@ export class SyncChecker {
       })
     }
 
+    // Make sure nodes aren't running too far ahead of altruists
+    if (currentBlockHeight + syncCheckOptions.allowance > altruistBlockHeight) {
```

Comment:
this is a personal pet peeve but on comparisons like this I recommend to put parentheses on the operations that must be grouped, on this case the sum, as denotes better how should be performed regardless of the operations order of the language, even though is kinda obvious in this case